source/_integrations/html5.markdown

```diff
@@ -311,7 +311,7 @@ You will receive an event named `html5_notification.closed` when the notificatio
 
 ### Making notifications work with NGINX proxy
 
-If you use [NGINX](/ecosystem/nginx/) as a proxy with authentication in front of your Home Assistant instance, you may have trouble with receiving events back to Home Assistant. It's because of authentication token that cannot be passed through the proxy.
+If you use [NGINX](/docs/ecosystem/nginx/) as a proxy with authentication in front of your Home Assistant instance, you may have trouble with receiving events back to Home Assistant. It's because of authentication token that cannot be passed through the proxy.
```

Comment:
Let's remove the link, as it is not part of our documentation anymore. Furthermore, this would apply to any NGINX reverse proxy set up, not just the one we used to document.

```suggestion
If you use NGINX as a proxy with authentication in front of your Home Assistant instance, you may have trouble with receiving events back to Home Assistant. It's because of an authentication token that cannot be passed through the proxy.
```